microsoftgraph/microsoft-graph-explorer
# Canary environment

## Overview

Changes the endpoint when the mode is set to canary

## Testing Instructions

* Run graph explorer
* Add **?mode=canary** to the url and hit enter
* Sidebar shoes that canary mode is activated
* Check to see that the url has changed. Incase it hasn't, hit refresh
* To disable this mode, hit the disable canary version on the sidebar
